versions.properties

```diff
@@ -164,3 +164,5 @@ version.net.sourceforge.pmd..pmd-java=version.net.sourceforge.pmd..pmd-core
 version.net.sourceforge.pmd..pmd-scala=version.net.sourceforge.pmd..pmd-core
 
 version.net.sourceforge.pmd..pmd-kotlin=version.net.sourceforge.pmd..pmd-core
+
+version.it.unibo.scafi..scafi-core_2.13=0.3.3
```

Comment:
There should also be one line deleted with the old scafi name

Reply:
Right